README.md

```diff
@@ -2,3 +2,58 @@
 Website for coordinating rehabilitation of people affected in the 2018 Kerala Floods
 
 We're so glad to see the PRs, will merge after releasing critical features
+=======
+# Kerala Rescue
+
+Website for coordinating rehabilitation of people affected in the 2018 Kerala Floods
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+You will need to have following softwares in your system
+
+- Python 3
+- Postgres
```

Comment:
Can't we simply use SQLite for development?

Reply:
We can, but I would personally like to maintain consistency with production over here hence Postgres.